trace/elasticsearch/elasticsearch.go

```diff
@@ -0,0 +1,161 @@
+package elasticsearch
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"runtime"
+	"strings"
+
+	"github.com/beatlabs/patron/trace"
+	"github.com/elastic/go-elasticsearch/v6"
+	"github.com/elastic/go-elasticsearch/v6/esapi"
+	"github.com/elastic/go-elasticsearch/v6/estransport"
+	"github.com/opentracing/opentracing-go"
+)
+
+const (
+	// If no addresses are passed, hit localhost by default
+	defaultURL = "http://localhost:9200"
```

Comment:
For some reasons you have 8 chars tab space...
you dont run gofmt?
something with your IDE?
The standard is 4 chars.